content/references/http-websocket-apis/public-api-methods/clio-methods/server_info-clio.md

```diff
@@ -482,6 +482,7 @@ The `info` object may have some arrangement of the following fields:
 | `Field`                             | Type            | Description          |
 |:------------------------------------|:----------------|:---------------------|
 | `complete_ledgers`                  | String          | Range expression indicating the sequence numbers of the ledger versions the local `rippled` has in its database. This may be a disjoint sequence such as `24900901-24900984,24901116-24901158`. If the server does not have any complete ledgers (for example, it recently started syncing with the network), this is the string `empty`. |
+| `counters`                          | Object          | This is present only if the client connects to the `Clio` server over `localhost`.
 | `rpc`                               | Object array    | Stats on each RPC handled by the Clip server since startup. |
```

Comment:
`rpc` and `subscriptions` are nested inside `counters`, so also only appear if the client connects over localhost